eng/pipelines/official.yml

```diff
@@ -107,12 +107,37 @@ extends:
       - template: /eng/pipelines/templates/jobs/workload-build.yml@self
         parameters:
           stabilizePackageVersion: ${{ parameters.stabilizePackageVersion }}
-          publishToFeed: ${{ parameters.publishToFeed }}
-          feedForPublishing: ${{ parameters.feedForPublishing }}
           createVSInsertion: ${{ parameters.createVSInsertion }}
           usePreComponentsForVSInsertion: ${{ parameters.usePreComponentsForVSInsertion }}
           includeNonShippingWorkloads: ${{ parameters.includeNonShippingWorkloads }}
           vsTopicBranch: ${{ parameters.vsTopicBranch }}
           workloadDropNames: ${{ parameters.workloadDropNames }}
           primaryVsInsertionBranches: ${{ parameters.primaryVsInsertionBranches }}
           secondaryVsInsertionBranches: ${{ parameters.secondaryVsInsertionBranches }}
+    - stage: Publish to feed
+      displayName: Publish
```

Comment:
This seems inverted. Should likely be the `stage` of `Publish` and the `displayName` of `Publish to feed`. But I'm not even certain that makes sense. Likely, your `deployment` job should have a `displayName` of `Publish to feed` because that's what the job does, not what the feed will do overall. This stage should also publish to NuGet, so having the job be `Publish to feed` and then there would be another job of `Publish to NuGet`. That seems to be better to me.